### PR TITLE
temporary files for reviews, not to be merged

### PR DESCRIPTION
**Pull request by janrygl**
We will discuss at school.

### CODE REVIEW

**Comment by janrygl** [comment]
@Kajakr @elakop If you have already written a review, please click "Finish review button".
If not, could you code review one of the files in this pull request, please?

File: reviews/beautify_text.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+def beautify(text):
+    mapping = {'A': '𝔄', 'B': '𝔅', 'C': 'ℭ', 'D': '𝔇', 'E': '𝔈', 'F': '𝔉', 'G': '𝔊', 'H': 'ℌ', 'I': 'ℑ', 'J': '𝔍', 'K': '𝔎', 'L': '𝔏', 'M': '𝔐', 'N': '𝔑', 'O': '𝔒', 'P': '𝔓', 'Q': '𝔔', 'R': 'ℜ', 'S': '𝔖', 'T': '𝔗', 'U': '𝔘', 'V': '𝔙', 'W': '𝔚', 'X': '𝔛', 'Y': '𝔜', 'Z': 'ℨ', 'a': '𝔞', 'b': '𝔟', 'c': '𝔠', 'd': '𝔡', 'e': '𝔢', 'f': '𝔣', 'g': '𝔤', 'h': '𝔥', 'i': '𝔦', 'j': '𝔧', 'k': '𝔨', 'l': '𝔩', 'm': '𝔪', 'n': '𝔫', 'o': '𝔬', 'p': '𝔭', 'q': '𝔮', 'r': '𝔯', 's': '𝔰', 't': '𝔱', 'u': '𝔲', 'v': '𝔳', 'w': '𝔴', 'x': '𝔵', 'y': '𝔶', 'z': '𝔷'}
+    return ''.join(mapping.get(char, char) for char in text)
+
+
+def convert(text): return beautify(text)
+
+
+def main(text): return convert(text)
```
Comment (Kajakr) [inline reviews/beautify_text.py:11]: the convert function is unnecessary

File: reviews/beautify_text.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import argparse
+
+def beautify(text):
```
Comment (Kajakr) [inline reviews/beautify_text.py:3]: I am not able to run the code - I guess that it is a  mistake with encoding

File: reviews/randomize_text.py
Comment (elakop) [inline reviews/randomize_text.py:1]: 1. missing #comments of this code
2. random shuffle:
 Using "random.shuffle" on chunks of text separated by `\b` can cause punctuation to appear in unexpected places.
Using "re.findall(r'\b\w+\b|[^\w\s]', text)" shuffles words and not individual characters
4. Mixing up words:
It is better to mix the words first and then connect them with spaces. This preserves sentence readability better than the original method
5. Missing arguments:
 If the user does not provide any text, the code should inform him.

image

File: reviews/beautify_text.py (hunk 2 above)
Comment (janrygl) [reply reviews/beautify_text.py:3]: Characters in mapping are UTF-8. Python3 is using UTF-8 as well, so if the file is saved as UTF-8, everything is OK.
Unfortunately, Windows still do not use UTF-8.
So you are right, on Windows, code doesn't work without handling encoding.

File: reviews/beautify_text.py (hunk 1 above)
Comment (janrygl) [reply reviews/beautify_text.py:11]: You are right. Also, this code formatting is discouraged.

File: reviews/beautify_text.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    mapping = {'A': '𝔄', 'B': '𝔅', 'C': 'ℭ', 'D': '𝔇', 'E': '𝔈', 'F': '𝔉', 'G': '𝔊', 'H': 'ℌ', 'I': 'ℑ', 'J': '𝔍', 'K': '𝔎', 'L': '𝔏', 'M': '𝔐', 'N': '𝔑', 'O': '𝔒', 'P': '𝔓', 'Q': '𝔔', 'R': 'ℜ', 'S': '𝔖', 'T': '𝔗', 'U': '𝔘', 'V': '𝔙', 'W': '𝔚', 'X': '𝔛', 'Y': '𝔜', 'Z': 'ℨ', 'a': '𝔞', 'b': '𝔟', 'c': '𝔠', 'd': '𝔡', 'e': '𝔢', 'f': '𝔣', 'g': '𝔤', 'h': '𝔥', 'i': '𝔦', 'j': '𝔧', 'k': '𝔨', 'l': '𝔩', 'm': '𝔪', 'n': '𝔫', 'o': '𝔬', 'p': '𝔭', 'q': '𝔮', 'r': '𝔯', 's': '𝔰', 't': '𝔱', 'u': '𝔲', 'v': '𝔳', 'w': '𝔴', 'x': '𝔵', 'y': '𝔶', 'z': '𝔷'}
+    return ''.join(mapping.get(char, char) for char in text)
```
Comment (janrygl) [inline reviews/beautify_text.py:5]: @Kajakr We also don't like such long lines, such text should be formatted or better, moved to txt file.

File: reviews/beautify_text.py (hunk 1 above)
Comment (janrygl) [reply reviews/beautify_text.py:11]: @Kajakr Well done.

File: reviews/randomize_text.py
Comment (janrygl) [reply reviews/randomize_text.py:1]: @elakop 
1. Actually, it is better to use docstrings (`""" """`) than comments (`#`).
2. You are right, original code is not working properly
3. I don't get the point: original code is respecting the order you mentioned.
4. Good point.

Code review covers all pains of the code, well done. I suggest to use multiple runs of LLM to double check if they are not hallucinating. But this review caught the main issue and that is the most important.

**Pull request closed without merging**